tools/debian_buildenv.sh

```diff
@@ -12,7 +12,7 @@ case "$1" in
     setup)
         source /etc/lsb-release 2>/dev/null
         case "${DISTRIB_CODENAME}" in
-            focal|jammy|bullseye) # <= Ubuntu 22.04.5 LTS
+            focal|jammy|bullseye|victoria|vera|vanessa|virginia) # <= Ubuntu 22.04.5 LTS
```

Comment:
Isn't it be possible to detect the qt version number instead? Maintaining a list of third party distribution codenames seems to be a very dirty workaround.

Reply:
Certainly it is. It was just easier to add these names and extend the otherwise working solution.

Reply:
I guess we don't really need to maintain that list anyway as this is only a list of old distros, so we won't have new one. This was more of an oversight, right?

Reply:
At least it fixes the reported issue and can be removed once these distros are obsolete.